products/azure-devops.md

```diff
@@ -1,62 +1,83 @@
 ---
 title: Azure DevOps
+category: server-app
+iconSlug: azuredevops
 permalink: /azure-devops
 alternate_urls:
 -   /tfs
 -   /team-foundation-server
-iconSlug: azuredevops
-category: server-app
 releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Azure%20DevOps
 activeSupportColumn: true
 releaseDateColumn: true
+
 releases:
+-   releaseCycle: "2022"
+    releaseDate: 2022-12-06
+    support: 2028-01-11
+    eol: 2023-01-11
```

Comment:
Are these 2 dates flipped?

![image](https://user-images.githubusercontent.com/584253/208383974-358ef1ef-db4e-4f11-8e28-c2d4de87c205.png)


```suggestion
    support: 2023-01-11
    eol: 2028-01-11
```

Reply:
Nope, EOL date should be 2033, not 2023 (see https://learn.microsoft.com/en-us/lifecycle/products/?terms=Azure%20DevOps)

Reply:
Fixed.